src/Web/Internal/FormUrlEncoded.hs

```diff
@@ -413,6 +417,8 @@ class FromForm a where
 
 instance FromForm Form where fromForm = pure
 
+instance FromForm () where fromForm _ = Right ()
```

Comment:
FromForm Void should fail.

Reply:
There is no `Void` here, this is `()`.

Reply:
There should be void too.

Reply:
Perhaps you meant to add an instance for `Void` that fails?

Reply:
Oh, I wrote the comment before I saw yours. On it!

Reply:
Done in: https://github.com/Daniel-Diaz/http-api-data/commit/14a5f5385fe2152e9a191c4aa1882cf44a9234a8